spring/src/main/kotlin/com/claytw/context/annotations/ComponentScan.kt

```diff
@@ -0,0 +1,5 @@
+package com.claytw.context.annotations
+
+@Target(AnnotationTarget.CLASS)
+@Retention
+annotation class ComponentScan(val path: String = "")
```

Comment:
What's this empty string used for?

Reply:
just be friendly to the programmer, if no path value was provided, the class package would be the root for scan